recipes/libtiff/all/conanfile.py

```diff
@@ -95,18 +96,17 @@ def _patch_sources(self):
                                   r'WINDOWS_EXPORT_ALL_SYMBOLS ON)')
 
             if self.settings.os == "Windows" and self.settings.compiler != "Visual Studio":
-                tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeListsOriginal.txt"),
+                tools.replace_in_file(cmakefile,
                                       "find_library(M_LIBRARY m)",
                                       "if (NOT MINGW)\n  find_library(M_LIBRARY m)\nendif()")
                 if self.version == '4.0.8':
                     # only one occurence must be patched. fixed in 4.0.9
-                    tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeListsOriginal.txt"),
-                                          "if (UNIX)",
-                                          "if (UNIX OR MINGW)")
+                    tools.replace_in_file(cmakefile, "if (UNIX)", "if (UNIX OR MINGW)")
 
         tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeListsOriginal.txt"),
                               "add_subdirectory(tools)\nadd_subdirectory(test)\nadd_subdirectory(contrib)\nadd_subdirectory(build)\n"
                               "add_subdirectory(man)\nadd_subdirectory(html)", "")
+        tools.replace_in_file(cmakefile, "LIBLZMA_LIBRARIES", "CONAN_LIBS_XZ_UTILS")
```

Comment:
maybe we can use generator (e.g. cmake_find_package)?

Reply:
Yes, sounds better.

Reply:
It's already there: https://github.com/conan-io/conan-center-index/pull/1200/files/c02276a547a1a5ff41b991575981626ca9aa2439#diff-bb572994c5caf9c21e2cf972a752ee0cR15

Reply:
@SSE4 it was the case-sensitive name, the find package provided by Conan contains LibLZMA_LIBRARIES instead of LIBLZMA_LIBRARIES, which is expected by libtiff

Reply:
do we need to update `cpp_info.names` for LZMA?

Reply:
It would be great, but I think only components can do that